src/main/kotlin/no/nav/dagpenger/streams/Topics.kt

```diff
@@ -46,6 +46,12 @@ object Topics {
         keySerde = strings,
         valueSerde = packetSerde
     )
+
+    val DAGPENGER_BEHOV_PACKET_EVENT_Q2 = Topic(
```

Comment:
kunne eventuelt lagt dette i konfig for hvert repo. ala joark mottak
https://github.com/navikt/dagpenger-joark-mottak/blob/master/src/main/kotlin/no/nav/dagpenger/joark/mottak/Configuration.kt#L89
?

Reply:
ja like greit